wit_dashboard/wit-dashboard.html

```diff
@@ -2684,94 +2689,109 @@ <h2>Create similarity feature</h2>
                         </div>
                       </div>
                       <iron-collapse opened="{{featureValueThreshold.opened}}">
-                        <div class="perf-table-entry-expanded flex-row-reverse">
+                        <div class="perf-table-entry-expanded">
```

Comment:
nit: is there a reason we removed flex-row-reverse from here but kept it in line 2896?

Reply:
added back in for consistency. this div only has one child so the rule has no effect currently, but should keep it in so its consistent across model types in case we add in more child elements here.